docs/diagrams/dev-guide/ArchitectureDiagram.puml

```diff
@@ -2,7 +2,7 @@
 !include <office/Concepts/globe_internet>
 !include <office/Concepts/documents>
 !include <office/Users/user>
-!include style.puml
+!include ../style.puml
```

Comment:
Can i clarify what does this do

Reply:
style.puml indicates attributes related to how the diagram would looks like visually. Like how a class/package is drawn. It comes with the original AB3. It's just that because we shifted the directory of the puml files, we need to change the directory to correctly locate style.puml

Reply:
ohhh thx